core/nodejsActionBase/app.js

```diff
@@ -1,96 +0,0 @@
-/*
```

Comment:
removing this file doesn't require updates to the gradle builds (for local build/dev) eg https://github.com/nimbella/openwhisk-runtime-nodejs/blob/a822e9ba7ef1bcabe0d376d2fa998cc95ef5f729/core/nodejs14Action/build.gradle#L45

Reply:
Is anything using the gradle builds?

Reply:
i have and maybe thinking we update the travis jobs as GHA to publish these images to dockerhub again... could defer it i don't object.

Reply:
Imma try

Reply:
Done!